.github/workflows/ci.yaml

```diff
@@ -16,7 +16,7 @@ env:
   CARGO_TERM_COLOR: always
   RUSTUP_MAX_RETRIES: 10
   PACKAGE_NAME: ruff
-  PYTHON_VERSION: "3.7" # to build abi3 wheels
+  PYTHON_VERSION: "3.11" # to build abi3 wheels
```

Comment:
I thought maybe we _had_ to use 3.7 for abi3 or something, but I don't really remember or know what I'm talking about. @konstin would know.

Reply:
hm

> PyO3 has feature flags abi3-py37, abi3-py38, abi3-py39 etc. to set the minimum required Python version for your abi3 wheel. For example, if you set the abi3-py37 feature, your extension wheel can be used on all Python 3 versions from Python 3.7 and up
>
> PyO3 is only able to link your extension module to abi3 version up to and including your host Python version. E.g., if you set abi3-py38 and try to compile the crate with a host of Python 3.7, the build will fail.

https://pyo3.rs/v0.19.1/building_and_distribution#py_limited_apiabi3

Looks safe to me?

Reply:
We should be able to test this easily enough locally as well

Reply:
Nice, yeah, that seems fine. It also may not even matter since we're only shipping a binary executable, which IIUC doesn't need to adhere to the ABI anyway. But maybe worth letting @konstin confirm.

Reply:
we're building a binary and not a library so this is safe :)